source/enums/SubtitleSelection.bs

```diff
@@ -0,0 +1,4 @@
+enum SubtitleSelection
+    notset = -2
+    none = -1
```

Comment:
```suggestion
    NOTSET = -2
    NONE = -1
```

Reply:
All other enums are uppercase. Since we're now exposing the subtitle selection enum (which I believe was our first ever enum added to the code), let's update it to match the style of the others.

The references in code should also be updated 👍🏻